Makefile

```diff
@@ -213,6 +213,7 @@ bam_markdup.o: bam_markdup.c config.h $(htslib_thread_pool_h) $(htslib_sam_h) $(
 tmp_file.o: tmp_file.c config.h $(tmp_file_h) $(htslib_sam_h)
 bam_ampliconclip.o: bam_ampliconclip.c config.h $(htslib_thread_pool_h) $(sam_opts_h) $(htslib_hts_h) $(htslib_hfile_h) $(htslib_kstring_h) $(htslib_sam_h) $(samtools_h) bam_ampliconclip.h
 bam_samples.o: bam_samples.c config.h $(htslib_hts_h) $(htslib_sam_h) $(htslib_faidx_h) $(htslib_khash_h) $(htslib_kseq_h) $(samtools_h)
+reset.o: reset.c $(samtools_h) $(htslib_hts_h)
```

Comment:
This needs more dependencies adding.  From the source I see:

    #include "samtools.h"
    #include "htslib/sam.h"
    #include "sam_opts.h"
    #include "htslib/thread_pool.h"
    #include "htslib/khash.h"

Reply:
Done.